modals/Overlay/index.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var React = require('react');
+var Icon  = require('../../atoms/Icon');
+var orginalOverflowY;
+var body;
+var document = document || false;
+
+module.exports = React.createClass({
+  displayName: 'Overlay',
+
+  getDefaultProps: function() {
+    return {
+      open: false
+    }
+  },
+
+  componentWillMount: function() {
+    if(document) {
+      body = document.getElementsByTagName('body')[0];
+      orginalOverflowY = body.style.overflowY;
+    }
+  },
+
+  render: function() {
+    var props = this.props;
+    if (!props.open) {
+      if(document) { body.style.overflowY = orginalOverflowY; }
+      return null;
+    }
+
+    if(document) { body.style.overflowY = 'hidden'; }
```

Comment:
The Overlay we implemented in the widget library has been developed to accommodate all sorts of modal content, including long scrollable lists (such as search results), and to ensure the Overlay works regardless of where it's called from. This implementation will have issues if position:relative|absolute|fixed are used anywhere in the DOM hierarchy above it, because it renders the Overlay element as a child node of whatever needs it.

I can show you what I'm doing for the aggregate search, and shoot you the code to implement it for this wizard.